jsBasics/task4.js

```diff
@@ -0,0 +1,17 @@
+//4.1
+const π = Math.PI;
+const radius = 10;
+const square = π * radius^2;
```

Comment:
^ - is not an operator you need
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR

either use ** or Math.pow